### DISCUSSION TITLE
Using certificates in the Keychain

### DISCUSSION THREAD

**Discussion by Oldes** [airsdk/Adobe-Runtime-Support#3599] [Q&A] [answered]
Hi,

is it now possible to directly reference the macOS Keychain in the adt command or one still must use certificate exported to a `.p12` file?

**Comment by ajwfrost** [comment, marked as answer]
Yes that should be possible with a signing section similar to:
```
-storetype KeychainStore -alias "Developer ID Application"
```
Sometimes I think I've found the full alias is required in order for it to find it.. other times it seems it just needs enough text to be unique within your Keychain store..

thanks
